frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss

```diff
@@ -34,11 +34,14 @@
 }
 
 .co-status-card__alert-item {
-  align-items: center;
   display: flex;
   padding-bottom: 0.5rem;
   padding-left: var(--pf-c-card--child--PaddingLeft);
   padding-right: var(--pf-c-card--child--PaddingRight);
+
+  .co-dashboard-icon {
```

Comment:
We should give this a separate class name like `.co-status-card__alert-item-icon` instead of using nesting.